Fix/up ci agent disk space

simonsarginson opened the pull request:
#### Description
upped disk space for ci agents from to 1000 gb

m-samiec commented:
What was the default size?

mironec commented on .buildkite/premerge.steps.yaml:
```diff
@@ -35,6 +35,7 @@ windows: &windows
     - "permission_set=builder"
     - "scaler_version=2"
     - "queue=${CI_WINDOWS_BUILDER_QUEUE:-v4-20-11-18-224740-bk17641-0c4125be-d}"
+    - "boot_disk_size_gb=1000"
```
Okay, this is gonna sound like bad CI design, but this will not actually increase the disk size for our agents.
You will instead have to adjust it here: `UnrealGDK\ci\gdk_build.template.steps.yaml`. The reason being is that the above machine template is only used for our slack notification step, not for the actual building step, that lives in the above file. This is due to us generating build steps dynamically. We could move the file into the .buildkite folder, I guess, since that would make it more discoverable, but kind of worried that will mess up something in buildkite.

simonsarginson replied:
The fact that this is for slack only is pretty well hidden and I think also counter the design of premerge.steps.yaml as indicated by EV to me. Out of scope to address but very confusing.

simonsarginson commented:
@m-samiec Default size is 500 gb as I found out from the buildkite artefacts. As @mironec mentions above the reason the default is not apparent in the file above is because I added it to the wrong file.. Fixed now by moving it into the correct yaml file.

mironec reviewed (approved):
But what will it cost? :(
I am always too cautious about making CI machines better, so approving.
